libcaf_core/biicode.conf

```diff
@@ -0,0 +1,42 @@
+# Biicode configuration file
+
+[requirements]
+    # Blocks and versions this block depends on e.g.
+    # user/depblock1: 3
+    # user2/depblock2(track) @tag
+
+[parent]
+	outro56/libcaf_core: 1
```

Comment:
I should probably remove the parent section here since it points to my published feed in biicode. It should be empty or point to your own feed published feed for libcaf